src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -129,6 +149,11 @@ private void Dispose(bool disposing)
                 _readerCancellationSource.Cancel();
                 _readerCancellationSource.Dispose();
             }
+
+            if (_process != null)
+            {
+                _process.Dispose();
+            }
```

Comment:
```suggestion
            _process?.Dispose();
```

Reply:
Aahh  :(  Have a habit of Java for sure will change in upcoming PRs as I have to additional connect tests